interface/ceed-operator.c

```diff
@@ -61,7 +62,9 @@ static int CeedOperatorCheckField(Ceed ceed, CeedQFunctionField qf_field, CeedEl
     }
     CeedCall(CeedBasisGetDimension(b, &dim));
     CeedCall(CeedBasisGetNumComponents(b, &num_comp));
-    CeedCall(CeedBasisGetNumQuadratureComponents(b, &Q_comp));
+    CeedCall(CeedBasisGetFESpace(b, &fe_space));
```

Comment:
Same here, would be good to actually use something more general like `Q_comp` that was here. And new variables should follow the naming contentions of 1) snake case and 2) using full words as much as reasonable

Reply:
Resolved in [063bea1](https://github.com/CEED/libCEED/pull/1156/commits/063bea108b882e83a5ae894d5a49e974101cbd30).